drivers/i3c/master/mipi-i3c-hci/core.c

```diff
@@ -203,7 +203,7 @@ static void aspeed_i3c_of_populate_bus_timing(struct i3c_hci *hci, struct device
 
 	/* Workaround . Need to remove once hardware support is available
 	   for internal LDO powerup support is available. */
-	if (!i3c_device_power) {
+	if (!i3c_device_power && hci->master.bus.context == I3C_BUS_CONTEXT_JESD403) {
```

Comment:
since this workaround , can you add note saying this also should reverted as part of workaround removal.